ReactNativeClient/lib/WebDavApi.js

```diff
@@ -17,6 +21,21 @@ class WebDavApi {
 		this.logger_ = new Logger();
 		this.options_ = options;
 		this.lastRequests_ = [];
+		if (shim.isLinux()) { 
+			if (Setting.value('sync.6.path').startsWith("https") || Setting.value('sync.5.path').startsWith("https")) {
```

Comment:
No Setting here please. This is a generic class that shouldn't have back references.